time_advance.m

```diff
@@ -66,6 +66,7 @@
     %     k3 = invMatLHS * (k3 + rhs3);
     k3 = ALHS \ (k3 + rhs3);
 else
+     % Captain! take off boundary condition vectors as a test:
```

Comment:
what?

Reply:
A semi coherent narration of some test I was doing. I think I was checking to see if results agreed before incorporating the boundary conditions